app/aws/s3.py

```diff
@@ -85,28 +79,3 @@ def remove_contact_list_from_s3(service_id, contact_list_id):
 def remove_s3_object(bucket_name, object_key, access_key, secret_key, region):
     obj = get_s3_object(bucket_name, object_key, access_key, secret_key, region)
     return obj.delete()
-
-
-def get_list_of_files_by_suffix(
```

Comment:
this wasn't being used anywhere (maybe it was letters related?) so it got the axe

Reply:
Oh yeah, that seems very possible.